cms/templates/js/add-xblock-component-menu-problem.underscore

```diff
@@ -1,3 +1,14 @@
+<% 
+    problem_names = ["Blank Common Problem", "Blank Advanced Problem", 
+        "Custom Javascript Display and Grading", "Custom Python-Evaluated Input", 
+        "Drag and Drop", "Molecular Structure", "Problem with Adaptive Hint"] 
```

Comment:
I really don't think this is the right way to go about this. Why can't the names be scraped and translated from the Python files, like the other phrases (such as "Math Expression Input") are?